File: sct_source/SourceCode/API_SoCongThuong/Models/ListOfKeyEnergyUsersModel.cs
namespace API_SoCongThuong.Models
{
    public class ListOfKeyEnergyUsersModel
    {
        public Guid ListOfKeyEnergyUsersId { get; set; }
        public Guid BusinessId { get; set; }
        public string? BusinessName { get; set; }
        public string Address { get; set; } = null!;
        public int? Date { get; set; }
        public DateTime? DateOnly { get; set; }
        public string Link { get; set; } = null!;
        public bool IsDel { get; set; }
        public Guid? Profession { get; set; }
        public string? ManufactProfession { get; set; }
        public decimal EnergyConsumption { get; set; } = 0;
        public string? Note { get; set; }
        public string DistrictName { get; set; } = "";
        public string ProfessionName { get; set; } = "";
        public Guid? District { get; set; } = Guid.Empty;
        public string Decision { get; set; } = "";
    }
}

File: sct_source/SourceCode/FE_SoCongThuong/Angular/src/app/modules/app-pages/list-of-key-energy-users-page/_models/list-of-key-energy-users.model.ts
import { BaseModel } from '../../../../_metronic/shared/crud-table/models/base.model';

export interface ListOfKeyEnergyUsersModel extends BaseModel {
  listOfKeyEnergyUsersId: string;
  businessId: string;
  address: string;
  date: any;
  link: string;
  profession: string;
  manufactProfession: string;
  energyConsumption: number;
  note: string;
  district: string;
  decision: string;
}
